source/Octopus.Tentacle.Upgrader/ServiceBouncer.cs

```diff
@@ -6,6 +6,7 @@
 using System.ServiceProcess;
 using System.Threading;
 using Microsoft.Win32;
+#pragma warning disable CA1416
```

Comment:
Does this not get handled by the global suppression?

Reply:
Thanks, good pickup. I've applied the CA1416 suppression to the whole project, since this project is only for Windows, and that's what the warning was for.

Reply:
_It wasn't being suppressed before as the global suppressions were only being done for certain members._